src/css/base.scss

```diff
@@ -11,57 +11,66 @@ body {
   background-color: $light-blue;
   background-image: url(../images/login-background.jpg);
   color: $eggplant;
-  margin: 0;
-  padding: 0;
+  @include no-margin-padding;
   height: 100vh;
 }
 
 header {
-  width: 100%;
-  height: 30px;
   background-color: transparent;
+  @include basic-flex(row nowrap)
+  height: 15%;
   margin: 0;
-  padding: 1em;
-  text-align: left;
+  padding: 1em 1em 1em 4%;
   position: fixed;
+  text-align: left;
+  width: 100%;
 }
 
 main {
+  align-items: center;
+  @include basic-flex(row wrap)
   height: 100%;
-  display: flex;
-  flex-flow: row wrap;
   justify-content: space-around;
-  align-items: center;
 }
 
 h1 {
-  padding: 0;
-  margin: 0;
   color: $eggplant;
+  font-size: 2em;
+  @include no-margin-padding;
+}
+
+h3 {
+  margin-top: 0px;
+}
+
+.compass {
+  height: 2.3em;
+  margin: 0 5px;
+}
+
+.pin {
+  height: 1em;
 }
 
 .login-form {
-  display: flex;
-  flex-flow: column nowrap;
-  width: 300px;
-  height: 350px;
-  border: 2px solid $clay;
-  border-radius: 10px;
-  padding: 1em;
   align-items: center;
-  justify-content: space-around;
   background-color: $transparent-white;
+  @include clay-border;
+  @include basic-flex(column nowrap)
```

Comment:
Nice use of mixins...you've eliminated several lines of repetitive code with this basic-flex one!